### PR TITLE
chore: update eslint configuration to avoid lint errors in json files

### PR DESCRIPTION
**Pull request by Dunqing**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [x] Other (about what?)

### 💡 Background and solution

fix 
image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix lint errors in JSON file       |
| 🇨🇳 Chinese |    修复 json 文件出现 lint error      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 1162bf7</samp>

Update ESLint rules for JSON files. Add a new configuration block in `.eslintrc.js` to disable some rules that are not applicable for JSON files.

### <samp>🤖 Generated by Copilot at 1162bf7</samp>

*  Add ESLint configuration for JSON files ([link](https://github.com/ant-design/ant-design/pull/42895/files?diff=unified&w=0#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R129-R135))

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -126,6 +126,13 @@ module.exports = {
         'react/no-danger': 0,
       },
     },
+    {
+      files: ['**/*.json'],
+      rules: {
+        'no-unused-expressions': 0,
+        'comma-dangle': 0,
+      },
+    },
```
Comment (afc163) [inline .eslintrc.js:135]: 我看现在 lint ci 没报错

Comment (Dunqing) [reply .eslintrc.js:135]: ci 不校验 json 文件

Comment (afc163) [reply .eslintrc.js:135]: 所以问题是啥，我本地 vscode 也没红线报错。

图片

Comment (Dunqing) [reply .eslintrc.js:135]: 你 vscode 有装 eslint 拓展吗？

Comment (afc163) [reply .eslintrc.js:135]: 图片

Comment (Dunqing) [reply .eslintrc.js:135]: 因为我的 eslint 配置里会校验 json 文件所以能复现出来，这里应该是规则本身有问题，不能因为我允许校验 json 文件就报错

**Review by afc163 — APPROVED** [review]

**Pull request merged**